src/styles/imageEditor.scss

```diff
@@ -14,10 +14,10 @@
   }
 
   &__gallery-actions {
-    @apply flex justify-between;
+    @apply flex justify-between w-galleryImage;
 
     &__button {
-      @apply text-whpptLightMode;
+      @apply text-whpptLightMode underline;
```

Comment:
Can we have the hover remove the underline. Really just for some hover interaction

Reply:
done.